integreat_cms/cms/templates/contacts/contact_form.html

```diff
@@ -0,0 +1,145 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load widget_tweaks %}
+{% block content %}
+    {% with request.region.default_language as current_language %}
+        <form id="content_form"
+              method="post"
+              enctype="multipart/form-data"
+              data-unsaved-warning
+              {% if contact_form.disabled %}data-disable-poi-query{% endif %}>
+            {% csrf_token %}
+            <div class="flex justify-between mb-4">
+                <h1 class="heading">
+                    {% if contact_form.instance.id %}
+                        {% translate "Edit contact" %}
+                    {% else %}
+                        {% translate "Create new contact" %}
+                    {% endif %}
+                </h1>
+                <div class="flex flex-wrap justify-between gap-4">
+                    {% if perms.cms.change_contact and not contact_form.instance.archived %}
+                        <div class="flex flex-wrap gap-4">
+                            <button class="btn">
+                                {% translate "Save" %}
+                            </button>
+                        </div>
```

Comment:
Should we differentiate here between save and update? Since this is what we currently do for abstract content translations

Reply:
What I thought was that it might make sense to label the button "Update" if the user is editing an existing contact and "Save" (or maybe "Create") if the user is creating a new contact